src/core/utils.ts

```diff
@@ -191,3 +191,35 @@ export function onDOMReady(fn: () => void) {
         document.addEventListener('DOMContentLoaded', fn);
     }
 }
+
+export function isMobile(): boolean {
+    return /android|iPhone|iPad|iPod|mobile/i.test(navigator.userAgent);
+}
+
+export function configHasDDLFeature(config: any) {
+    return (
+        config &&
+        Array.isArray(config.features) &&
+        config.features.includes(SDKFeature.DDL)
+    );
+}
+
+export function deferPromptActivation(
```

Comment:
Filing cabinet type thing, this bleeds prompt stuff into the `core` module, can/should sit in `prompts`?

Reply:
Yeah got same note down here too for this 👍